### ISSUE TITLE
structcheck false positives with embedded structs

### ISSUE THREAD

**Issue by tamird** [opennota/check#11]
```go
package main

type Foo struct {
	a int
}

type Bar struct {
	Foo
}

func (b Bar) A() int {
	return b.a
}

func main() {
	Bar{}.A()
}
```
```
$ structcheck ./...
ex.go:4: Foo.a
```

**Comment by opennota** [comment]
This is already mentioned in the README under "Known limitations."
